proto/regen/ecocredit/v1/tx.proto

```diff
@@ -273,12 +323,106 @@ message MsgCreateProjectResponse {
   string project_id = 1;
 }
 
+// MsgCreateUnregisteredProject is the Msg/CreateUnregisteredProject request type.
+message MsgCreateUnregisteredProject {
+  option (cosmos.msg.v1.signer) = "admin";
+
+  // admin is the address of the account creating the project that will become
+  // the admin of the project upon creation.
+  string admin = 1;
+
+  // metadata is any arbitrary string with a maximum length of 256 characters
+  // that includes or references metadata to attach to the project.
+  string metadata = 2;
+
+  // jurisdiction is the jurisdiction of the project. A jurisdiction has with
+  // the format: <country-code>[-<sub-national-code>[ <postal-code>]]
+  // The country-code must be 2 alphabetic characters, the sub-national-code
+  // can be 1-3 alphanumeric characters, and the postal-code can be up to 64
+  // alphanumeric characters. Only the country-code is required, while the
+  // sub-national-code and postal-code are optional and can be added for
+  // increased precision.
+  string jurisdiction = 3;
+
+  // reference_id is any arbitrary string used to reference the project with a
+  // maximum length of 32 characters.
+  string reference_id = 4;
+
+  // fee is the project creation fee. An equal fee is required if the project
+  // creation fee parameter is set. The provided fee can be greater than the
+  // parameter, but only the amount in the parameter will be charged.
+  //
+  // Since Revision 3
+  cosmos.base.v1beta1.Coin fee = 5;
+}
+
+// MsgCreateUnregisteredProjectResponse is the Msg/CreateUnregisteredProject response type.
+message MsgCreateUnregisteredProjectResponse {
+  // project_id is the unique identifier of the project.
+  string project_id = 1;
+}
```

Comment:
How will project IDs be generated? Will they be auto-incrementing IDs unique across all credit classes?

Reply:
Auto-increment across all projects independent of credit class.

Reply:
No, all project IDs would be stable and uncorrelated with credit classes